recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,5 @@
 arm_variant_type: # [aarch64]
   - sbsa          # [aarch64]
+conda_glibc_ver:
+  - 2.17          # [not aarch64]
```

Comment:
I feel like this is the kind of thing we should probably define in aggregate?

Reply:
it seems to be defined in package cbcs for everything that uses it (various compiler related packages). But yeah I agree. Would prefer this not to be blocked on that though, and move them all out at some later opportunity. Concretely, I'd generate a jira ticket for it, and if needed an epic for cbc reviewing. Would you be ok with that?

Reply:
You know what, I'm even wondering we should shouldn't just do `>=2.17` on both x86_64 and aarch64. If something is compiled with glibc 2.17, it will work on anything greater than that version. Like I don't know why they hard pin glibc there.

Reply:
Do you know why they specify compilers and sysroot in this feedstock anyway? It's just binary repackaging.

Reply:
I had this exact question with https://github.com/AnacondaRecipes/file-feedstock/pull/1 yesterday and the only thing I can think of is that it's to make the DSO check happy.

Reply:
is that for linkage checks?

Reply:
right yeah

Reply:
I asked that [here](https://github.com/conda-forge/cuda-nvcc-impl-feedstock/issues/14)

Reply:
I haven't had a reply and this is in line with other feedstocks ([example](https://github.com/AnacondaRecipes/gcc_toolchain-feedstock/blob/525581cbf9edd348209e0bf8c56be267e86ee969/recipe/meta.yaml#L145)). Would you be ok with keeping it in? Seems less risky if we don't know the reason. Would like to get the PR moving along. Don't think it's critical to unpin.